# Simulation parameters
num_points: 500 # Number of ground observations
boundary_radius: 50 # Radius in meters of circular test area
ground_height_max: 0.0 # max height of random ground points (0.0 is global flat-Earth assumption)
cluster_radius: 2.0 # Radius in meters for clustering ground points into flight waypoints
num_poses: 1500 # Number of camera poses, or "images"
flight_AGL: 10.0 # Flight altitude (AGL) in meters

# Camera intrinsics
image_width: 1920 # Horizontal image resolution in pixels
image_height: 1200 # Vertical image resolution in pixels
fx: 4244.23289558 # Horizontal focal-length component in pixels
fy: 4241.64222077 # Vertical focal-length component in pixels
cx: 928.08491883 # Horizontal coordinate of focal origin in pixels
cy: 578.38700405 # Vertical coordinate of focal origin in pixels

# Noise parameters
sigma_x: 0.014 # std of x translation estimation in meters
sigma_y: 0.014 # std of y translation estimation in meters
sigma_z: 0.014   # std of z translation estimation in meters

# Inertialsense IMX5-INS (GNSS-RTK, 1 m baseline)
sigma_roll: 0.0006981317  # std of roll attitude estimation in radians
sigma_pitch: 0.0006981317 # std of pitch attitude estimation in radians
sigma_yaw: 0.002268928     # std of yaw attitude estimation in radians

# sigma_roll: 0.004363323  # std of x attitude estimation in radians
# sigma_pitch: 0.004363323 # std of y attitude estimation in radians
# sigma_yaw: 0.0139626     # std of z attitude estimation in radians

#sigma_roll: 0.0  # std of x attitude estimation in radians
#sigma_pitch: 0.0 # std of y attitude estimation in radians
#sigma_yaw: 0.0     # std of z attitude estimation in radians

sigma_RPE_x: 0.413286 # std of horizontal pixel reprojection error
sigma_RPE_y: 0.343144 # std of vertical pixel reprojection error

# ortho-synthetic filtering parameters
eps: 0.02     # epsilon DBSCAN parameter (size of neighborhood in meters)
min_points: 5 # minimun neighborhood denisty threshold
